d2compiler/compile.go

```diff
@@ -820,6 +822,24 @@ func (c *compiler) compileEdgeMap(edge *d2graph.Edge, m *d2ir.Map) {
 	}
 }
 
+func (c *compiler) compileEdgeLinks(edge *d2graph.Edge, m *d2ir.Map) {
```

Comment:
i think this function should just be a switch case in the function `compile.go`'s `compileReserved`